<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>506</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>921</width>
    <height>506</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ADSI Decryptor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitSidebar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <widget class="QFrame" name="sidebarPanel">
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>240</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="accessibleName">
        <string>sidebar</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="btnImageLoad">
          <property name="text">
           <string>Load Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnImageExport">
          <property name="text">
           <string>Verify</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>267</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnSetting">
          <property name="text">
           <string>Setting</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="clientAreaPanel">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="components::ImagePreview" name="labImagePreview">
          <property name="minimumSize">
           <size>
            <width>1</width>
            <height>1</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>components::ImagePreview</class>
   <extends>QLabel</extends>
   <header>components/ImagePreview.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnSetting</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSettingClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>652</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnImageLoad</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onLoadImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>595</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnImageExport</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onVerifyImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>572</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSettingClicked()</slot>
  <slot>onLoadImage()</slot>
  <slot>onVerifyImage()</slot>
 </slots>
</ui>
